packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -53,7 +53,7 @@ abstract contract Sphinx {
      * @dev The configuration options for the user's project. This variable must have `internal`
      *      visibility so that the user can set fields on it.
      */
-    SphinxConfig public sphinxConfig;
+    UserSphinxConfig public sphinxConfig;
```

Comment:
I've renamed `SphinxConfig` to `UserSphinxConfig` in the Solidity plugin since it now has significantly less fields (only `projectName`, `mainnets`, and `testnets`). I've also introduced a new type `InternalSphinxConfig` which is exactly the same as the old `SphinxConfig` type. `InternalSphinxConfig` is used within our plugin and is included in the `DeploymentConfig` object so that this update is not a breaking change to the executor.